<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgRectangleConstraints</class>
 <widget class="QWidget" name="WdgRectangleConstraints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Size</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <property name="spacing">
      <number>7</number>
     </property>
     <item column="3" row="1">
      <widget class="QPushButton" name="lockHeightButton">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="3" row="2">
      <widget class="QPushButton" name="lockRatioButton">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="3" row="0">
      <widget class="QPushButton" name="lockWidthButton">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="2" row="1">
      <widget class="KisIntParseSpinBox" name="intHeight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Height</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="2" row="2">
      <widget class="KisDoubleParseSpinBox" name="doubleRatio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Aspect ratio</string>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item column="2" row="0">
      <widget class="KisIntParseSpinBox" name="intWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Width</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item column="1" row="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ratio:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item column="0" row="0">
      <widget class="QLabel" name="lblRoundCornersX">
       <property name="text">
        <string>Round X:</string>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="KisIntParseSpinBox" name="intRoundCornersX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Horizontal radius of rectangle corners</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item column="2" row="0" rowspan="2">
      <widget class="KoAspectButton" name="cornersAspectButton" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item column="0" row="1">
      <widget class="QLabel" name="lblRoundCornersY">
       <property name="text">
        <string>Round Y:</string>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="KisIntParseSpinBox" name="intRoundCornersY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Vertical radius of rectangle corners</string>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>187</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleParseSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>kis_double_parse_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>